tests/callbacks/test_RecorderListener.cpp

```diff
@@ -0,0 +1,186 @@
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+#include "Kokkos_Core.hpp"
+
+#include "kokkos-utils/impl/type_traits.hpp"
+
+#include "kokkos-utils/callbacks/EventInProfileSectionRegexMatcher.hpp"
+#include "kokkos-utils/callbacks/EventRegexMatcher.hpp"
+#include "kokkos-utils/callbacks/RecorderListener.hpp"
+
+#include "tests/callbacks/Helpers.hpp"
+#include "tests/callbacks/TestWorkload.hpp"
+
+/**
+ * @addtogroup unittests
+ *
+ * @c Kokkos callback recorder listener
+ * ------------------------------------
+ *
+ * This group of tests check the behavior of @ref Kokkos::utils::callbacks::RecorderListener.
+ */
+
+using execution_space = Kokkos::DefaultExecutionSpace;
+
+namespace Kokkos::utils::tests::callbacks
+{
+
+using namespace Kokkos::utils::callbacks;
+
+//! Listener to record events that occur in a profile section.
+using event_in_profile_section_recorder_t = RecorderListener<EventInProfileSectionRegexMatcher, EventTypeList>;
+
+class RecorderListenerTest : public ManagerTestFixture
+{
+public:
+    void SetUp() override {
+        this->exec = Kokkos::Experimental::partition_space(execution_space{}, 1)[0];
+    }
+
+protected:
+    execution_space exec {};
+};
+
+//! @test Check traits of @ref Kokkos::utils::callbacks::RecorderListener.
+TEST_F(RecorderListenerTest, traits)
```

Comment:
No need for the fixture ;)
```suggestion
TEST(RecorderListener, traits)
```

Reply:
OK! Good catch! Done for all.